flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/hybrid/HsMemoryDataSpiller.java

```diff
@@ -60,13 +64,17 @@ public class HsMemoryDataSpiller implements AutoCloseable {
     /** File channel to write data. */
     private final FileChannel dataFileChannel;
 
+    private final BufferCompressor bufferCompressor;
```

Comment:
Should mark `@Nullable`. Same for the corresponding constructor parameter.

Reply:
marked.